### PR TITLE
Create Release.md

### PR DESCRIPTION
**Pull request by connyduck**
So everyone knows the things I do before a release and I could maybe hand this over to someone else.

### CODE REVIEW

**Review by nikclayton — COMMENTED** [review]
Just added some suggestions for future work. I'm not suggesting they need to be implemented before this PR is merged.

File: Release.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Merge the latest Weblate translations
+- Check all the translations (Android Studio shows warnings on problems). Sometimes translators add faulty translations that would crash Tusky in this language, e.g. wrong number of formatting parameters. In this case it is usually easiest to just delete the string. [Example cleanup](https://github.com/tuskyapp/Tusky/commit/feaea70af418c77178985144a2d01a8e97725dfd).
+- Update `versionCode` and `versionName` in `app/build.gradle`
+- Add a new short changelog under `fastlane/metadata/android/en-US/changelogs`. Use the next versionCode as the filename. This is so translators on Weblate have the duration of the beta to translate the changelog and F-Droid users will see it in their language on the release. If another beta is released, the changelogs have to be renamed. Note that changelogs shouldn't be over 500 characters or F-Droid will truncate them.
+- Build the app as apk and as app bundle.
+- Do a quick sanity check to make sure the build doesn't crash. Also install it over the last release to make sure the database migrations are correct.
```
Comment (charlag) [inline Release.md:15]: Can we please not use "sanity"?

File: Release.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Upload the release to the Open Testing track on Google Play.
+- Announce the release
+
+## Full release
+
+- Make sure all new features are well tested by beta users and all issues addressed as good as possible. Check GitHub issues, Google Play crash reports, messages on `@Tusky@mastodon.social`, #Tusky hashtag.
+- (optionally merge new translations and check them)
+- Update `versionCode` and `versionName` in `app/build.gradle`
+- Build the app as apk and as app bundle.
+- Do a quick sanity check to make sure the build doesn't crash. Also install it over the last release to make sure the database migrations are correct.
```
Comment (charlag) [inline Release.md:32]: Same

**Review by Tak — APPROVED** [review]

File: Release.md (hunk 1 above)
Comment (connyduck) [reply Release.md:15]: yes, very good point

File: Release.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Merge `develop` into `main`
+- Create a new [GitHub release](https://github.com/tuskyapp/Tusky/releases).
+  - Tag the head of `main`.
+  - Resuse the changelog from the beta release, or create a new one if this is only a minor release.
+  - Attach the apk, adb and mapping.txt files to the release
+- (F-Droid will automatically detect and build the release)
+- Upload the release to the Production track on Google Play.
+- update the download link on the homepage ([repo](https://github.com/tuskyapp/tuskyapp.github.io))
+- Announce the release
```
[the remark this answers is not in the document]
Comment (connyduck) [reply Release.md:41]: I'd leave that to decide for the releasing person

File: Release.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+## Beta
+
+- Make sure all new features are well tested by Nightly users and all issues addressed as good as possible. Check GitHub issues, Google Play crash reports, messages on `@Tusky@mastodon.social`, emails on `tusky@connyduck.at`, #Tusky hashtag.
+- Merge the latest Weblate translations
+- Check all the translations (Android Studio shows warnings on problems). Sometimes translators add faulty translations that would crash Tusky in this language, e.g. wrong number of formatting parameters. In this case it is usually easiest to just delete the string. [Example cleanup](https://github.com/tuskyapp/Tusky/commit/feaea70af418c77178985144a2d01a8e97725dfd).
+- Update `versionCode` and `versionName` in `app/build.gradle`
+- Add a new short changelog under `fastlane/metadata/android/en-US/changelogs`. Use the next versionCode as the filename. This is so translators on Weblate have the duration of the beta to translate the changelog and F-Droid users will see it in their language on the release. If another beta is released, the changelogs have to be renamed. Note that changelogs shouldn't be over 500 characters or F-Droid will truncate them.
+- Build the app as apk and as app bundle.
```
[the remark this answers is not in the document]
Comment (connyduck) [reply Release.md:14]: There are multiple ways to do it and it should be automated with bitrise anyway so I'll not bother with a detailed description for now.

**Pull request merged**